homeassistant/components/blueprint/websocket_api.py

```diff
@@ -84,3 +87,82 @@ async def ws_import_blueprint(hass, connection, msg):
             },
         },
     )
+
+
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "blueprint/save",
+        vol.Required("domain"): cv.string,
+        vol.Required("filename"): cv.string,
```

Comment:
This should be `path`. 
```suggestion
        vol.Required("path"): cv.path,
```